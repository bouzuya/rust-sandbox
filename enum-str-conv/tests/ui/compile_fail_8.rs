fn main() {
    fn unknown_fn(_s: String) {
        unreachable!()
    }

    #[derive(enum_str_conv::EnumStrConv)]
    #[enum_str_conv(error = (), unknown = unknown_fn)]
    enum MyEnum {
        #[enum_str_conv(error = (), str = "apple")]
        A,
    }
}
