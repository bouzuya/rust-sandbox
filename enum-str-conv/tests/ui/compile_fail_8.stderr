error: unknown argument: #[enum_str_conv(str = ...)]
 --> tests/ui/compile_fail_8.rs:9:25
  |
9 |         #[enum_str_conv(error = (), str = "apple")]
  |                         ^^^^^^^^^^
